src/lib/chat/matrix-client.ts

```diff
@@ -297,9 +297,18 @@ export class MatrixClient implements IChatClient {
     switch (event.type) {
       case EventType.RoomMessage:
         return mapMatrixMessage(event, this.matrix);
+
       case CustomEventType.USER_JOINED_INVITER_ON_ZERO:
       case EventType.RoomCreate:
         return mapEventToAdminMessage(event);
+
+      case EventType.RoomMember:
+        if (event.content.membership === 'leave') {
```

Comment:
In the next admin message PR I will change `leave` to use the `MembershipStateType.Leave`